openfl/interface/cli.py

```diff
@@ -173,6 +176,25 @@ def error_handler(error):
     raise error
 
 
+def review_plan_callback(file_name, file_path):
+    """Review plan callback for Director and Envoy."""
+    echo(style(
+        f'Please review the contents of {file_name} before proceeding...',
+        fg='green',
+        bold=True))
+    time.sleep(3)
```

Comment:
Please add a comment or named variable for understanding what 3 seconds mean and why it is 3 seconds.

Reply:
That was added for the director/envoy to read the instruction to review the contents of the plan before the plan gets flashed.